### PR TITLE
add BoosterPack iface

### PR DESCRIPTION
**Pull request by ajm01**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  <version>1.0-SNAPSHOT</version>
+
+  <name>boost-maven-common</name>
+  <!-- FIXME change it to the project's website -->
+  <url>http://www.example.com</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
```
Comment (scottkurz) [inline pom.xml]: Target Java 8

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        <plugin>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <version>3.7.0</version>
+        </plugin>
+        <plugin>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <version>2.20.1</version>
+        </plugin>
+        <plugin>
+          <artifactId>maven-jar-plugin</artifactId>
```
Comment (scottkurz) [inline pom.xml]: Curious where you got these versions from?   No objection..just wondering.

File: src/main/java/boost/project/BoosterPackConfigurator.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package boost.project;
+
+import org.w3c.dom.Document;
+
+public interface BoosterPackConfigurator {
```
Comment (scottkurz) [inline src/main/java/boost/project/BoosterPackConfigurator.java]: Maybe add some Javadoc

File: src/main/java/boost/project/BoosterPackConfigurator.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package boost.project;
```
Comment (scottkurz) [inline src/main/java/boost/project/BoosterPackConfigurator.java]: Add EPL copyright

File: pom.xml (hunk 2 above)
Comment (ajm01) [reply pom.xml]: i used a maven archetype goal to create this at one point - and those were the defaults that got genned

**Pull request merged**